fordel/src/main/java/no/nav/ung/domenetjenester/personhendelser/HåndterUngSakHendelseTask.java

```diff
@@ -18,9 +20,6 @@
 import java.time.LocalDateTime;
 import java.util.List;
 
-import static no.nav.ung.domenetjenester.personhendelser.HendelseMapper.fraJson;
-import static no.nav.ung.domenetjenester.personhendelser.HendelseMapper.toJson;
-
 @ApplicationScoped
 @ProsessTask(HåndterUngSakHendelseTask.TASKNAME)
 public class HåndterUngSakHendelseTask implements ProsessTaskHandler {
```

Comment:
Burde kanskje endre navn til noe som indikerer at det bare er hendlser for personinformasjon/PDL som håndteres her? Nå tilsier navnet at alle hendelser håndteres her

Reply:
Godt poeng 👍